torch/csrc/jit/api/function_impl.h

```diff
@@ -100,6 +100,12 @@ struct TORCH_API GraphFunction : public Function {
     return true;
   }
 
+  void clear_optimized_graphs() {
```

Comment:
is this the right way of clearing out an array ? im not that well-versed on this c++ idiom cc @jjsjann123

Reply:
I suppose std::fill might be better?

Reply:
std::array::fill. So I suppose something like
`optimized_graphs_.fill(c10::nullopt);`